newsroom/notifications/module.py

```diff
@@ -25,6 +25,7 @@
                 name="user_created",
                 keys=[("user", 1), ("_created", -1)],
                 collation={"locale": "en", "strength": 2},
+                unique=False,
```

Comment:
why is this change to the index here?

Reply:
In the new code, we're defaulting to `unique:True`, and after auditing the problems we've had with indexes and build issues and the code, I made these changes to make sure those that are meant to be unique are unique